### PR TITLE
python3Packages.python-telegram-bot: 13.0 -> 13.3

### PR DESCRIPTION
**Pull request by pingiun**
###### Motivation for this change
This is the latest version

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

My own python bot is running this currently

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115287` at 9d2811db run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/d5abb332503961d4823165e33ef0ed80">python38Packages.python-telegram-bot</a></li></li>
    <li><a href="https://gist.github.com/2ac0f27f67017e252ef0ea9529de677a">python39Packages.python-telegram-bot</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`lgpl3` is a deprecated license, check if project uses `lgpl3Plus` or `lgpl3Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/python-telegram-bot/default.nix:40:5:
```
   |
40 |     license = licenses.lgpl3;
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/python-telegram-bot/default.nix [hunk 1]
```diff
@@ -7,21 +7,22 @@
...
 , pytest
+, APScheduler
 , isPy3k
 }:
 
 buildPythonPackage rec {
   pname = "python-telegram-bot";
-  version = "13.0";
+  version = "13.3";
   disabled = !isPy3k;
```
Comment (fabaff) [inline pkgs/development/python-modules/python-telegram-bot/default.nix:17]: There is a [restriction](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/setup.py#L107) on the supported Python 3 release.

```suggestion
  disabled = pythonOlder "3.6";
```

**Review by fabaff — COMMENTED** [review]
Could you please update to license while you are on it? Looks like [`lgpl3Only`](https://github.com/python-telegram-bot/python-telegram-bot#license).

The upstream repo contains [tests](https://github.com/python-telegram-bot/python-telegram-bot/tree/master/tests). Just something we should keep in mind. I doubt that they would work without tweaking.

File: pkgs/development/python-modules/python-telegram-bot/default.nix (hunk 1 above)
Comment (veprbl) [reply pkgs/development/python-modules/python-telegram-bot/default.nix:17]: We don't ship python 3 older than 3.6

**Comment by veprbl** [comment]
@GrahamcOfBorg build python38Packages.python-telegram-bot python39Packages.python-telegram-bot

File: pkgs/development/python-modules/python-telegram-bot/default.nix [hunk 2]
```diff
@@ -7,25 +7,29 @@
...
+    hash = "sha256-dw1sGfdeUw3n9qh4TsBpRdqEvNI0SnKTK4wqBaeM1CE=";
   };
 
   checkInputs = [ pytest ];
-  propagatedBuildInputs = [ certifi future urllib3 tornado decorator ];
+  propagatedBuildInputs = [ certifi future urllib3 tornado decorator APScheduler ];
 
   # --with-upstream-urllib3 is not working properly
   postPatch = ''
     rm -rf telegram/vendor
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-telegram-bot/default.nix:29]:
```suggestion
    rm -r telegram/vendor
```

File: pkgs/development/python-modules/python-telegram-bot/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/python-telegram-bot/default.nix:17]: Not sure how we handle this but I have no problem with adding disableds.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 115287` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python38Packages.python-telegram-bot</li>
<li>python39Packages.python-telegram-bot</li>
</ul>
</details>

**Pull request merged**